lib/NP/Model/DnsRoot.pm

```diff
@@ -158,11 +158,35 @@ sub populate_country_zones {
         if (my $entries = $zone->active_servers('v6')) {
             @$entries = shuffle(@$entries);
 
-            # for now just put all IPv6 servers in the '2' zone
+            # add all servers to the non-numbered "NTP" zone
+            (my $pgeodns_group = "${name}") =~ s/\.$//;
+            push @{$data->{$name}->{aaaa}}, $_ for @$entries;
+
+            # Historically, only '2' has had ipv6 records, so we ALSO put all
+            # of the records there, for anyone who is relying on 2 having a
+            # large number of entries (Who would do such a crazy thing?)
             (my $pgeodns_group = "2.${name}") =~ s/\.$//;
             push @{$data->{$pgeodns_group}->{aaaa}}, $_ for @$entries;
-        }
 
+            # Smear the rest of the IPv6 records over the remaining zones,
```

Comment:
I'd rather have IPv4 and IPv6 work the same; except .. again, we need a toggle to not swap the behavior for the "vendor zones".  (Some of them might not handle IPv6 records and expect 0.x to not have them, for example, and many of them might not have contact information or even any active maintenance or support).